joplin/base/wagtail_hooks.py

```diff
@@ -47,6 +56,22 @@ def before_edit_page(request, page):
     print(f'BeforeEditHook {request.user.email} is in groups {[group.name for group in request.user.groups.all()]}')
 
 
+@hooks.register('construct_main_menu')
+def configure_main_menu(request, menu_items):
+    new_items = []
+    for item in menu_items:
+        if item.name in ('home', 'images'):
+            item.label = ''
+            new_items.append(item)
+    menu_items[:] = new_items
```

Comment:
Ignorant question. What is this syntax? Not sure how to Google it.

Reply:
It's slice syntax. It's so we set the original `menu_items` objects instead of replacing it with a new list.